runtime/compilers/rillv1/parse_metrics_view.go

```diff
@@ -6,29 +6,29 @@ import (
 	"fmt"
 	"strings"
 	"time"
+	// Load IANA time zone data
+	_ "time/tzdata"
```

Comment:
Lint error – anonymous imports should be in the third section

Reply:
Got it. My editor has different defaults. Will fix these.